test/twin/coercion_test.rb

```diff
@@ -44,6 +44,14 @@ class TwinWithSkipSetter < Disposable::Twin
       subject.band.label.value.must_equal "9999.99"
     end
 
+    it "NOT coerce twice on subclasses" do
+      subclassed = Class.new(TwinWithSkipSetter).new(album)
+      subclassed.released_at.must_equal "31/03/1981"
+
+      subclassed.released_at =  "12/12/1980"
```

Comment:
He, you get a `TypeError` because the value "31/03/1981" gets coerced twice, the first time with the string, the second time with a `DateTime`.